Fixtures/ClangModules/Cpp/Sources/cwrapper/cwapper.cpp

```diff
@@ -0,0 +1,7 @@
+#import "include/cwrapper.h"
+#import "cpplib.h"
+
+int cwrapperfive() {
```

Comment:
Thanks this totally fixed it!
@Danappelxx file name doesn't really matter but thanks :P